pallets/nfts/src/tests.rs

```diff
@@ -204,6 +206,7 @@ fn lifecycle_should_work() {
 			account(10),
 			default_item_config()
 		));
+		assert_eq!(AccountBalance::<Test>::get(0, account(10)), 1);
```

Comment:
Could we give those accounts variable names so that it is clear who is owner of a collection / item. I understand that this is how it is done in the tests for the original pallet but lets provide the tests for the dispatchables we changed or created with variable names.